src/app/components/showcase.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+
+export default function Showcase() {
+  return (
+    <div className="flex justify-center p-5 mt-[100vh] bg-black">
+      <div className="max-w-prose">
+        <h1 className="p-3 text-center font-serif text-2xl">{"Photography"}</h1>
+        <ShowcaseItem title="Sports">
+          This will become a description for sports photography
+        </ShowcaseItem>
+        <ShowcaseItem title="Automotive">
+          This will become a description for automotive photography
+        </ShowcaseItem>
+        <ShowcaseItem title="Landscapes">
+          This will become a description for landscape photography
+        </ShowcaseItem>
+
+        <h1 className="p-3 text-center font-serif text-2xl">{"Videos"}</h1>
+        <ShowcaseItem title="Mountain biking">
+          This will become a description for mountain bike videos
+        </ShowcaseItem>
+        <ShowcaseItem title="Automotive">
+          This will become a description for automotive videos
+        </ShowcaseItem>
+        <ShowcaseItem title="Skiing">
+          This will become a description for skiing videos
+        </ShowcaseItem>
+      </div>
+    </div>
+  )
+}
+
+interface ShowcaseItemProps {
```

Comment:
When trying react props, it's generally preferably to use `type` instead of `interface`. Both are pretty similar, however `type` offers a bit of an easier to use syntax when the types get complex.

Reply:
Oh great, thanks! This was my first time working with prop types and TypeScript, so `interface` is just what I found online. I'll work on updating this!

Reply:
Done in a recent commit!